### DISCUSSION TITLE
Tab Loading Error

### DISCUSSION THREAD

**Discussion by dcs3473** [jneilliii/OctoPrint-BedLevelVisualizer#417] [General]
When clicking on the Tab to open Bed Level Visualizer it takes sometimes 30-60 seconds to open.  There are many times when I get an error from the Browser (Tried Several) that this page is slowing your browser and to I want to Stop or Wait.  I can hit wait a couple times and the page will load.  After the tab is loaded everything works as expected.  No other tabs like Terminal, Control, etc cause this issue.

**Comment by jneilliii** [comment]
I noticed this yesterday and there was a discussion in the OctoPrint discord server about it [here](https://discord.com/channels/704958479194128507/708230829050036236/824761569341997096). For me it was the DuckDuckGo Privacy Extension for Chrome that was causing it. As soon as I disabled that extension it worked normally. I haven't had much time yet to investigate why. Another user mentioned that when they only had the BLV plugin installed it didn't do it, but haven't heard back yet of what the potential plugin conflict could be.
